src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -521,9 +542,83 @@ else if (job.getTree().hasLeaves() && job.getTree().isSlimeTree())
             }
             job.getTree().pollNextLeaf();
         }
+        else if (job.getTree().hasLeaves() && job.getTree().isNetherTree())
+        {
+            final BlockPos leaf = job.getTree().peekNextLeaf();
+            if (!mineBlock(leaf, workFrom))
+            {
+                return getState();
+            }
+            job.getTree().pollNextLeaf();
+        }
         return getState();
     }
 
+    /**
+     * Bonemeal fungi in netherTrees
+     */
+
+    private IAIState bonemealFungi()
+    {
+        for (final BlockPos pos : getNetherTrees())
+        {
+            final BlockState blockState = world.getBlockState(pos);
```

Comment:
Here we want to check first if the position is actually loaded, use the WorldUtil function for that